### PR TITLE
Parametrize active storage service config option

### PR DESCRIPTION
**Pull request by heitorado**
### Overview
The default storage is still Amazon S3 so current implementations do not break. The new environment variable `'STORAGE_SERVICE'`  can be used to choose other storage services specified on `storage.yml`.

### Further Explanation
I am proposing this change due to something I've run into while setting chaskiq on production: I've done a Caprover deploy and did not want to set up an AWS S3 account. I created a new custom image just to change the active storage service configuration to 'local'. When we parameterize the active storage service, it allows us to support multiple configurations on `storage.yml` and then passing the desired one via the `STORAGE_SERVICE` environment variable.

Plus, using the 'local' storage allows a cheaper and quicker production proof-of-concept to get things up and running with less dependencies and exploring what chaskiq has to offer. Then, changing the storage service afterwards is only a few environment variables away.

Hope this suggestion is relevant, I'm available to discuss it further if you wish.

### CODE REVIEW

**Comment by michelson** [comment]
Thank you for this PR,, @heitorado!. This is interesting, but not sure if we are going to merge this as is. The *local* storage is not intended for production deployment. So if we want to add parametrized options to the activestorage I would like to add some options for other storage providers too. We could work on adding this.

**Comment by heitorado** [comment]
Great! Sure thing @michelson, we can work on adding other storage providers, so we can have 3-4 possible values for this new env var. I'll go after the required parameters for them asap.

Out of curiosity, may I ask why local storage is not adequate for production deployment?

**Comment by michelson** [comment]
Hey @heitorado, thanks for your collaboration, It is greatly appreciated.

> Out of curiosity, may I ask why local storage is not adequate for production deployment?

I think this is because many state-of-the-art hosting services provide ephemeral disk storage, like Heroku or k8s. of course, you can do whatever you want, but I've read that using disk storage on those services is discouraged.

File: config/environments/production.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   # config.active_storage.service = :local
 
-  config.active_storage.service = :amazon
+  config.active_storage.service = ENV['STORAGE_SERVICE'].try(:to_sym) || :amazon
```
Comment (michelson) [inline config/environments/production.rb]:
```suggestion
  config.active_storage.service = Chaskiq::Config.fetch("STORAGE_SERVICE", "amazon").to_sym
```

**Review by michelson — APPROVED** [review]

**Comment by michelson** [comment]
already merged on another PR, thanks

**Pull request closed without merging**